docs/remote_inference_blueprints/gcp_vertexai_connector_embedding_blueprint.md

```diff
@@ -0,0 +1,267 @@
+```Response:
+{
+  "model_id": "pn8scY4B2QHLlv0i6LZB",
+  "task_type": "REGISTER_MODEL",
+  "function_name": "REMOTE",
+  "state": "COMPLETED",
+  "worker_node": [
+    "pQg-fbRpSm-x-V-0ICQpQw"
+  ],
+  "create_time": 1711295752216,
+  "last_update_time": 1711295752273,
+  "is_async": false
+}
+```
+
+## 5. Deploy the model
+
+```json
+POST /_plugins/_ml/models/pX8scY4B2QHLlv0i6LYZ/_deploy
+```
+Sample response:
+
+```Response:
+{
+  "task_id": "p38xcY4B2QHLlv0iSbb_",
+  "task_type": "DEPLOY_MODEL",
+  "status": "COMPLETED"
+}
+```
+
+## 6. Check model deployment
+```json
+GET /_plugins/_ml/tasks/pX8scY4B2QHLlv0i6LYZ
+```
+
+Sample response:
+```
+{
+  "model_id": "pn8scY4B2QHLlv0i6LZB",
+  "task_type": "REGISTER_MODEL",
+  "function_name": "REMOTE",
+  "state": "COMPLETED",
+  "worker_node": [
+    "pQg-fbRpSm-x-V-0ICQpQw"
+  ],
+  "create_time": 1711295752216,
+  "last_update_time": 1711295752273,
+  "is_async": false
+}
+```
+
+
+## 7. Test model inference
+
+```json
+POST /_plugins/_ml/models/pn8scY4B2QHLlv0i6LZB/_predict
+{
+  "parameters": {
+    "prompt": "Hello World form vertex AI!"
+  }
+}
+```
+
+Response:
+```json
+{
+  "inference_results": [
+    {
+      "output": [
+        {
+          "name": "response",
+          "dataAsMap": {
+            "predictions": [
+              {
+                "embeddings": {
+                  "statistics": {
+                    "truncated": false,
+                    "token_count": 7
+                  },
+                  "values": [
+                    0.006371465977281332,
+                    -0.03583695366978645,
+                    -0.009082107804715633,
+                    -0.018701229244470596,
+                    0.00281976698897779,
+                    -0.00454947492107749,
+                    0.012263650074601173,
+                    -0.011159989982843399,
+                    -0.01578657515347004,
+                    0.004389482084661722,
+                    0.0002078384131891653,
+                    -0.000520859903190285,
+                    0.03948991373181343,
+                    -0.057535942643880844,
+                    -0.005250570364296436,
+                    -0.02551070787012577,
+                    0.022418972104787827,
+                    0.029485566541552544,
+                    0.01078018918633461,
+                    -0.010538897477090359
+                    ....
+                  ]
+                }
+              }
+            ],
+            "metadata": {
+              "billableCharacterCount": 23
+            }
+          }
+        }
+      ],
+      "status_code": 200
+    }
+  ]
+}
+```
+
+
+## 8. Update the access token
+
+This requirement arises because the Vertex API can only be accessed using the GCP access token, which has a limited lifespan. Applications are responsible for refreshing this token as needed. 
+
+Note:  This update feature is only avaialable in 2.12 or later
+
+```json
+POST /_plugins/_ml/models/qwshEo4BP0vcvNchDtVA
```

Comment:
this is a different model id than above, if using the same model id would be more consistent

Reply:
Thanks @mingshl I missed it, I have corrected this, please review